public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -513,26 +569,30 @@ table
 .l-sub-section
   :marked
     Technically, Angular is matching the name to a directive [input](#inputs-outputs),
-    one of the property names listed in the directive’s `inputs` array or a property decorated with `@Input()`.
-    Such inputs map to the directive’s own properties.
+    one of the property names listed in the directive's `inputs` array or a property decorated with `@Input()`.
+    Such inputs map to the directive's own properties.
 :marked
   If the name fails to match a property of a known directive or element, Angular reports an “unknown directive” error.
 
   ### Avoid side effects
-  As we've already discussed, evaluation of a template expression should have no visible side effects. The expression language itself does its part to keep us safe. We can’t assign a value to anything in a property binding expression nor use the increment and decrement operators.
+  As mentioned previously, evaluation of a template expression should have no visible side effects. 
+  The expression language itself does its part to keep you safe. 
+  You can't assign a value to anything in a property binding expression nor use the increment and decrement operators.
 
-  Of course, our expression might invoke a property or method that has side effects. Angular has no way of knowing that or stopping us.
+  Of course, the expression might invoke a property or method that has side effects. Angular has no way of knowing that or stopping you.
 
-  The expression could call something like `getFoo()`. Only we know what `getFoo()` does.
-  If `getFoo()` changes something and we happen to be binding to that something, we risk an unpleasant experience. Angular may or may not display the changed value. Angular may detect the change and throw a warning error. Our general advice: stick to data properties and to methods that return values and do no more.
+  The expression could call something like `getFoo()`. Only directive know what `getFoo()` does.
```

Comment:
Oh my!  Some kind of search and replace changed "your" or "we" to "directive".  Seems to have been limited to this location. Fixed it.